content/en/docs/pipelines/installation/overview.md

```diff
@@ -49,6 +56,10 @@ Interfaces
   * Kubeflow Pipelines UI
   * Kubeflow Pipelines SDK
   * Kubeflow Pipelines API
+  * Kubeflow Pipelines endpoint is **only auto-configured** for GCP.
```

Comment:
> "  * Kubeflow Pipelines endpoint is **only auto-configured** for GCP."

Maybe: 
> "Kubeflow Pipelines endpoint is _currently only auto-configured_ for GCP.

Also, hasn't GCP been officially renamed to Google Cloud (without the "platform" part) as part of the rebrand?

Reply:
(The first point introduces "currently" because AWS and Azure, among others, may introduce that feature too in future.)